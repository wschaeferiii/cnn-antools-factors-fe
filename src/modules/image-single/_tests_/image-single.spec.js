import React from 'react';
import { shallow } from 'enzyme';
import ImageSingle from '../index';

const PROPS = {
    slug: '_2016_12_us_chicago_disappearing_front_porch_index_html'
}

describe('Image-Single UI Component', () => {
    it('should render', () => {
        const wrapper = shallow(<ImageSingle slug = {PROPS.slug} />);
        expect(wrapper).toMatchSnapshot();
    })
})
