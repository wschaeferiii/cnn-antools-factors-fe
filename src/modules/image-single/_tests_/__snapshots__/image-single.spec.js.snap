// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image-Single UI Component should render 1`] = `
<div
  className="foo"
>
  <FetchData
    url="http://data.cnn.com/jsonp/content/ksa/factors.json"
  />
</div>
`;
